custom/icds_reports/management/commands/sql_scripts/extract_valid_phone_count_grouped_by_district.sql

```diff
@@ -0,0 +1,108 @@
+SELECT 
+person_cases.district_id,
+person_cases.state_id,
+-- Pregnant Women
+SUM(
+    CASE 
+        WHEN is_pregnant = 1 
+            THEN 1 
+        ELSE 0
+    END) "pw_count",
+SUM(
+    CASE 
+        WHEN is_pregnant = 1 
+        AND phone_number similar to '91[6789][0-9]{9}' THEN 1 else 0
+    END) "pw_valid_phone_count",
+SUM(
+    CASE
+        WHEN is_pregnant = 1
+        AND (phone_number = '') IS NOT FALSE  THEN 1 ELSE 0
```

Comment:
does it also include `phone_number is null`?

Reply:
I actually used it by referring https://stackoverflow.com/questions/23766084/best-way-to-check-for-empty-or-null-value

I tested on my local machine it was working for null as well.
Do you think I should write an explicit statement for null?